### DISCUSSION TITLE
How to integrate with Python Web Frameworks (e.g., Flask, DJango, FastAPI).

### DISCUSSION THREAD

**Discussion by fpichardom** [sqlitecloud/discussions#26] [SDK] [answered]
The examples I have seen using Python are connecting via the Python SDK, but I don't understand how we can use the SDK to connect to SQLALCHEMY for Flask or the Django ORM, for example. I would like to know if that is possible already or if it will be included in the future.

**Comment by rajat709** [comment]
+1

**Comment by Baylexx** [comment]
Was wondering if there were any updates on this. My current flask app relies heavily on SQLALCHEMY.

**Comment by unatarajan** [comment, marked as answer]
Apologies for the delayed reply. Our team has been actively working to address this issue.

SQLAlchemy support is now available and documented in several locations:
- `sqlalchemy-sqlitecloud` package [README](https://github.com/sqlitecloud/sqlitecloud-py/tree/main/sqlalchemy-sqlitecloud)
- `sqlitecloud-py` [README](https://github.com/sqlitecloud/sqlitecloud-py?tab=readme-ov-file#sqlite-cloud-for-sqlalchemy-beta)
- [Quickstart](https://docs.sqlitecloud.io/docs/quick-start-sqlalchemy-orm) showing how to connect to a SQLite Cloud DB in FastAPI via SQLAlchemy

Please let us know if you experience any issues or have any further questions.

**Reply by unatarajan** [reply]
Also @fpichardom here is a [Quickstart](https://docs.sqlitecloud.io/docs/quick-start-django) explaining how to connect to SQLite Cloud in Django.

**Comment by fpichardom** [comment]
This is great news. Thanks so much for the hard work. This will surely allow me to better integrate sqlitecloud in my applications and workflows. Looking forward to start testing it now.
